src/components/ui/ComponentsLayout.vue

```diff
@@ -44,3 +47,11 @@ export default defineComponent({
   flex-direction: column;
 }
 </style>
+
+<FDRPlotly
```

Comment:
This should not be necessary and must be removed. Can you try if you can get the plot to be displayed without it? You can also open a PR to my FLASHViewer fork. Then we could troubleshoot it together.

Reply:
@t0mdavid-m I think a single plot would be sufficient—either the ECDF plot or the density distribution plot of target and decoy Qscores, with counts on the y-axis. What do you think?

Reply:
Up to you! We could always implement one plot and add the other plot later if necessary.

Did you try removing the hardcoded section in `ComponentsLayout.vue`?

Reply:
i will do it after lunch. and keep you updated if its done

Reply:
@t0mdavid-m the 

> This should not be necessary and must be removed. Can you try if you can get the plot to be displayed without it? You can also open a PR to my FLASHViewer fork. Then we could troubleshoot it together.

your suggestion is incorporated and the code block is removed and without it its also showing the fdr plot.